package-lock.json

```diff
@@ -243,12 +244,13 @@
       }
     },
     "node_modules/@next/swc-darwin-arm64": {
```

Comment:
Remove all this unwanted dependencies from PR, it was for your system and it shouldnt be required to everyone

Reply:
@vansh-codes can you specify which ones?

Reply:
the nextjs version one?

Reply:
swc-darwin-arm64
linux
node_modules/@next/swc-linux-arm64-gnu
msvc
and many more such linux and darwins dependencies

I guess you use linux thats why it was needed for you, but ig its not needed in the project, right?

Reply:
Not the nextjs one, the other ones in package-lock.json

Reply:
@vansh-codes i copied you current files, the only diff were of nextjs versions

Reply:
@Sourabh782 I am talking about `package-lock.json` and not `package.json`

Reply:
yup copied both

Reply:
![image](https://github.com/user-attachments/assets/cc7e6ec3-49b1-4c4b-aefc-e81cb303ab6f)

Reply:
is it still showing this? i actually just copied your package-lock.json and make a commit